packages/clerk-js/src/core/services/authentication/SessionCookieService.ts

```diff
@@ -95,7 +86,7 @@ export class AuthenticationService {
   }
 
   private setClientUatCookieForDevelopmentInstances() {
-    if (this.environment && this.environment.isDevelopmentOrStaging() && this.inCustomDevelopmentDomain()) {
+    if (this.environment?.isDevelopmentOrStaging() && this.inCustomDevelopmentDomain()) {
```

Comment:
[You've just increased the bundle size by ~34 bytes!](https://www.typescriptlang.org/play?ts=4.9.5#code/LAKAZgrgdgxgLgSwPZQARwKYGc4AoBuAhgDYBcqhUAngJSoDeoqqMKO62cqAvKkcQH4AdAAsMxYkgDcTFmy6YcAJh58SqAGQa1xUeMmgAvkA)

Reply:
I will revert this change. Should we generally use the previous syntax instead of the `?.` operator ?